inst/@sym/isinf.m

```diff
@@ -37,16 +37,11 @@
   if isscalar(x)
 
     cmd = { 'd = _ins[0]'
-            'if sympy.__version__ == "0.7.5":'
-            '    d = d.is_unbounded'
-            'else:'
-            '    d = d.is_infinite'
-            'if d == True:'
```

Comment:
not yet please,  There is a `drop_sympy075` branch that does this sort of thing but I wasn't going to merge it yet